src/main/kotlin/com/broeskamp/monorepo/gradle/plugin/kotlin/AndroidKotlinPlugin.kt

```diff
@@ -60,4 +68,25 @@ class AndroidKotlinPlugin : Plugin<Project> {
       }
     }
   }
+
+  protected fun Project.buildVersionCode(): Int = if (project.hasProperty("android.version.code")) {
+    project.property("android.version.code").toString().toInt()
+  } else {
+    val versionObject = Version.parse("$version")
+    var vc = versionObject.major * 100_000_000
+    vc += versionObject.minor * 100_000
+    vc += versionObject.patch * 100
```

Comment:
Hier könnten wir uns ja auch 2 stellen sparen und dann bei den 2 Zeilen vorher auch jeweils zwei stellen wegnehmen

Reply:
könnten wir, aber schau mal in die andere conversiation, ich denke wir haben genug platz für die 3-stelligkeit